zulipterminal/model.py

```diff
@@ -52,6 +52,9 @@
     'message': Message,
     'flags': List[str],
     'subject': str,
+    'property': str,
```

Comment:
The other sets of possible entries have 'headers' for each section, which we don't have for these three new ones; we can reformat or add to these existing comments too, if they're not clear enough.

Reply:
Oh! Nice catch. I missed the headers here. Note to add them in future.